docs/data-management/organizing-metrics/guardrails.md

```diff
@@ -2,18 +2,29 @@
 sidebar_position: 6
 ---
 
-# Guardrail metrics
+# Guardrail cutoffs
 
-A guardrail metric refers to a specific type of metric in Eppo that is set as a standard measurement for every experiment.
-By configuring a metric as a guardrail, it is automatically included in the analysis of all applicable experiments, ensuring consistent tracking and comparison of results.
-This approach enables you to establish a uniform measurement framework, allowing you to monitor and evaluate key performance indicators consistently across various tests, providing valuable insights and facilitating data-driven decision-making processes.
+A guardrail cutoff refers to a downside threshold that can be set on a metric. When the results for the metric breach this threshold, the metric will display a yellow warning state.
+
+Use a guardrail cutoff where a neutral result is okay but a downside risk should be avoided. They also enable non-inferiority testing when a metric with a guardrail is set as primary.
```

Comment:
```suggestion
Guardrail cut-off metrics are primarily useful to track metrics that an experiment is not supposed to positively impact, but you want to ensure that there is no negative effect either. A common use case is page load times: few experiments explicitly want to improve page load times, but likely most experiments want to avoid slowing down page load times. Guardrail cut-offs also enable non-inferiority testing when a metric with a guardrail is set as primary.
```